source/ranges.tex

```diff
@@ -10066,7 +10060,7 @@
   concept @\defexposconcept{zip-is-common}@ =                             // \expos
     (sizeof...(Rs) == 1 && (@\libconcept{common_range}@<Rs> && ...)) ||
     (!(@\libconcept{bidirectional_range}@<Rs> && ...) && (@\libconcept{common_range}@<Rs> && ...)) ||
-    ((@\libconcept{random_access_range}@<Rs> && ...) && (@\libconcept{sized_range}@<Rs> && ...));
+    (@\exposconcept{random-access-sized}@<Rs> && ...);
```

Comment:
Isn't this a breaking change?

Reply:
Yes, this change is semantics-changing: Previously, we did not check any `sized_range` constraint if a single `random_access_range` constraint was not satisfied, meaning hard errors in the `sized_range` check wouldn't fire in that  case.

Now they might fire.

Reply:
Thanks for the comment. Although I doubt any *reasonable* code would actually reflect such difference.

Reply:
The standard is not about reasonable code, but about possible code. Naturally, the absurd niches get 80% of the attention, in general, because getting them right is tricky.